### DISCUSSION TITLE
Android 4.4.2 webview issue

### DISCUSSION THREAD

**Discussion by AndreiGr** [thecowan/wallpanel-android#343] [Q&A]
Hi guys,

So I installed this app with the goal to use an old tablet as a dashboard for HA.
Everything works fine except HA is stuck on the initializing screen, which I read is because an outdated version on webview 4.4.2. Unfortunately, the webview on 4.4.2 cannot be updated separately.

However, the latest version of Chrome available for 4.4.2 seems to bring a newest webview, because in chrome everything works fine.

Do I have any chances to make Wallpanel work too? Or, is it possible for Wallpanel to bring with it a newer webview version, for the usage of older devices?

Thank you,
Andrei

**Comment by vrabi-cv** [comment]
I got probably same problem... Is there any way of using my old tablet?
![IMG_20221014_201221](https://user-images.githubusercontent.com/110232498/195913902-397d260e-4221-4303-8f71-4ed76ecf6c53.jpg)
